src/main/java/de/hysky/skyblocker/skyblock/FairySouls.java

```diff
@@ -152,12 +152,20 @@ private static void onChatMessage(Text text, boolean overlay) {
 
     private static void markClosestFairyFound() {
         if (!fairySoulsLoaded.isDone()) return;
+
         PlayerEntity player = MinecraftClient.getInstance().player;
         if (player == null) {
             LOGGER.warn("[Skyblocker] Failed to mark closest fairy soul as found because player is null");
             return;
         }
-        fairySouls.get(Utils.getLocationRaw()).stream()
+
+        Set<BlockPos> soulsAtLoc = fairySouls.get(Utils.getLocationRaw());
+        if (soulsAtLoc == null) {
+            LOGGER.warn("[Skyblocker] Failed to mark closest fairy soul as found because soulsAtLoc is null. NEU repo probably failed to load.");
+            return;
+        }
+
+        soulsAtLoc.stream()
```

Comment:
Try to keep the naming convention consistent within a file.
```suggestion
        Set<BlockPos> fairiesOnCurrentIsland = fairySouls.get(Utils.getLocationRaw());
        if (fairiesOnCurrentIsland == null) {
            LOGGER.warn("[Skyblocker] Failed to mark closest fairy soul as found because there are no fairy souls loaded on the current island. NEU repo probably failed to load.");
            return;
        }

        fairiesOnCurrentIsland.stream()
```